src/components/Dashboard/FutureEffectiveFiling.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div class="future-effective-filing body-2">
+    <h4>Future Effective Incorporation Date</h4>
+
+    <p>The incorporation date and time for {{entityName || 'unknown'}}
+      will be {{effectiveDateTime || 'unknown'}} Pacific Time.</p>
+    <p>If you wish to change the information in this application you must
+      contact Registry Staff to file a withdrawal. </p>
+    <p>Withdrawing this Incorporation Application will remove this application
+      and all associated information, and will incur a $20.00 fee. </p>
+
+    <ErrorContact />
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop } from 'vue-property-decorator'
+import { mapState } from 'vuex'
+import { ErrorContact } from '@/components/common'
+
+@Component({
+  computed: { ...mapState(['entityName']) },
+  components: { ErrorContact }
+})
+export default class FutureEffectiveFiling extends Vue {
```

Comment:
Very cool, is this little component what ties in with a Future effective filing in the Filing History List?

Reply:
Yes. ATM it's specific to IAs. It really cleaned up the Filings History List code:
```
          <template v-else-if="filing.isIaFutureEffective">
            <future-effective-filing :filing=filing />
            <v-divider class="mt-7 mb-5"></v-divider>
          </template>
```

Reply:
Awesome!